appserver/web/web-core/src/main/java/org/apache/catalina/core/StandardContextValve.java

```diff
@@ -79,7 +79,7 @@
  * @author Craig R. McClanahan
  * @version $Revision: 1.19 $ $Date: 2007/05/05 05:31:54 $
  */
-
+// Portions Copyright [2021] [Payara Foundation and/or its affiliates]
```

Comment:
There's already a portions copyright block above on line 58 - you can just change the 2019 to 2021 there.